danielMunteanu/src/main/java/danielCollectionTask/FilterByType.java

```diff
@@ -0,0 +1,21 @@
+package danielCollectionTask;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+class FilterByType {
+    public static List<Order> filterOrderByItemType(List<Order> orderList) {
```

Comment:
why not making this method more general?

Reply:
fixed in next commit

Reply:
Nothing has been changed. Could you take a look once again?